### ISSUE TITLE
Android: Open new georeferenced template

### ISSUE THREAD

**Issue by ollesmaps** [OpenOrienteering/mapper#1225]
### Steps to reproduce
At the moment you can use only desktop preadjusted templates.

### Expected behaviour
It would be nice if you could open new templates also on Android.
This could be only for georeferenced templates.

**Comment by dg0yt** [comment]
The problem is that Android doesn't use the file system in a traditional way, but is build around 'media' and 'content providers'. That's why we can't just use the traditional file dialog in the past. There are more options now, with new versions of Android and Qt, but it still needs work (and at least a first Mapper release with a new version of Qt).

**Comment by ollesmaps** [comment]
Thank you for explanation. It is not a preesing issue. It would be a nice feature though.

**Comment by mikaello** [comment]
I'm importing an [OCAD 10 file](http://www.ringerike-o-lag.net/testkart/2018_Rostjern/Ocad10/2018_Rostjern_10000_oc10.ocd) in the Android app, and I am getting this warning: `Could not load the coordinate reference system '63005'` on opening. Is this related to/same as this issue/feature request?

**Comment by dg0yt** [comment]
@mikaello No, it is not this issue. You are "opening the map", not "adding a new template" to an existing map. You can test the unstable desktop version if you want to know if this CRS is covered in the coming 0.9 releases where OCD file support is much improved.

**Comment by mikaello** [comment]
I see. Thanks for your fast reply!

> You can test the unstable desktop version if you want to know if this CRS is covered in the coming 0.9 releases where OCD file support is much improved.
That was a good tip, it worked in the unstable 0.9 release (on OSX).

**Comment by ollesmaps** [comment]
What about this workaround? 
1. Duplicate template (that is possible on Android).
2. Rename it (new feature).
3. New template appears.

Could that work in some stances? 
(for example if the original template has the same world file attached as duplicate)
